charts/index.yaml

```diff
@@ -65,34 +75,23 @@ entries:
     version: 0.1.4
   - apiVersion: v2
     appVersion: 68-8e12ab5
-    created: "2021-10-29T16:37:08.32663-04:00"
+    created: "2021-11-02T17:06:15.208782-04:00"
     description: Simple Cromwell deployment with in-memory database running jobs locally
     digest: 56c38a6b51039fe0ffa458db073272a616d3f3ae0fdf7eb8fb9079ee17604122
     name: cromwell
     type: application
     urls:
     - https://broadinstitute.github.io/cromwhelm/charts/cromwell-0.1.3.tgz
     version: 0.1.3
-  cromwellwithpostgress:
-  - apiVersion: v2
-    appVersion: 1.16.0
-    created: "2021-10-29T16:37:08.329601-04:00"
-    description: Simple Cromwell deployment with Postgres database
-    digest: ad93460102e6d790e3255981b1dc151b36ef578064eaff7950186cd6258f8a37
-    name: cromwellwithpostgress
-    type: application
-    urls:
-    - https://broadinstitute.github.io/cromwhelm/charts/cromwellwithpostgress-0.1.0.tgz
-    version: 0.1.0
```

Comment:
I was wondering why this block was removed? Do we not need it to run postgres?

Reply:
No, this was something I accidentally introduced because I was helping Adam to debug something. There's no checked in cromwellwithpostgress file, so this section shouldn't actually exist.

Reply:
And to be clear, the actual helm chart we are using (which now is using Postgres) is called simply "cromwell".